index.js

```diff
@@ -15,7 +15,7 @@ var enqueue =
 var createClass = (obj) => {
   var out = ""
 
-  if (typeof obj === "string") return obj
+  if (!obj || typeof obj === "string") return obj
```

Comment:
Why `!obj`? Do we need this now?

Reply:
Because how I use createClass further down, obj could be undefined (for vnodes that do not have a class prop)

Reply:
And why didn't we have that issue before?

Reply:
Because we had this function call inside an if-clause in patchProperty (see the diff in patchProperty), so it would always have some value.